misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -161,8 +161,8 @@ public void test(String fileName, String path) throws IOException {
                 "annot_decl_source_02.bal", "class_def_source_11.bal", "object_type_def_source_26.bal",
                 "ambiguity_source_32.bal", "service_object_constructor_source_01.bal",
                 "service_object_constructor_source_02.bal", "object_constructor_source_08.bal",
-                "annotations_source_01.bal", "do_stmt_source_08.bal", "func_params_source_10.bal",
-                "func_params_source_11.bal", "func_params_source_13.bal");
+                "annotations_source_01.bal", "do_stmt_source_08.bal", "func_params_source_14.bal",
+                "func_params_source_15.bal", "func_params_source_13.bal", "func_params_source_11.bal");
```

Comment:
As an example, when formatting the following code sample with syntax errors, 

```
function foo(int a, int b, *person) {
}
```

the appropriate formatting would be,

```
function foo(int a, int b, *person ) {
}
```
Please note the _space_ after **`*person`**.

Here even if multiple tokens are expected after `*person` by the parser, still only a single space should be present. 

So, to enable such negative test cases, the relevant parser test source files should be reformatted accordingly.

Reply:
Fixed.